src/lib.rs

```diff
@@ -688,6 +688,22 @@ impl UnixDatagram {
         })
     }
 
+    /// Creates a Unix Datagram socket which is connected to specified addresss
+    /// the socket is unnamed and similar to one created by `new()` except
+    /// it will send message to the specified addresss *by default*
+    pub fn connect<P: AsRef<Path>>(path: P) -> io::Result<UnixDatagram> {
```

Comment:
It seems like this should be an instance method that allows an existing socket to connect.